include/linux/soundwire/sdw.h

```diff
@@ -421,8 +422,12 @@ struct sdw_master_prop {
 	u32 err_threshold;
 	u32 mclk_freq;
 	bool hw_disabled;
+	u32 quirks;
 };
 
+#define SDW_MASTER_QUIRKS_CLEAR_INITIAL_CLASH	BIT(0)
+#define SDW_MASTER_QUIRKS_CLEAR_INITIAL_PARITY	BIT(1)
+
```

Comment:
@plbossart I think Vinod also asked adding comments for each quirk.

>  #define SDW_MASTER_QUIRKS_CLEAR_INITIAL_CLASH	BIT(0)
> +#define SDW_MASTER_QUIRKS_CLEAR_INITIAL_PARITY	BIT(1)

Why not add this quirk in patch 1..?

**Also add comments about each quirk, hopefully it wont be a big table**

Reply:
well this is technically patch 1 for the quirks, the previous patches are just to deal with rebases.

Also I added a slew of comments in the commit log, not sure what more I can add? Are you saying add those comments in the header file as well?

Reply:
> Also I added a slew of comments in the commit log, not sure what more I can add? Are you saying add those comments in the header file as well?

Yes, but lets check with Vinod.

Reply:
@plbossart Yes, Vinod confirmed that he prefer having comments in both commit log and the header file.

Reply:
ack, will do.

Reply:
updated @bardliao